### PR TITLE
FE-932 Integrate D1 troubleshooting URLs

### PR DESCRIPTION
**Pull request by PavlosTze**
## **Why?**
New URLs:
- For low battery in Devices List & Device Alerts - https://docs.weatherxm.com/wxm-devices/D1-bundle/D1-assemble#installing-batteries
- For D1 Troubleshooting in Device Details - https://docs.weatherxm.com/wxm-devices/D1-bundle/D1-troubleshooting

### **How?**
Added the "Low battery url" in Devices List & Alerts screen and the "Troubleshooting" URL in the Device Details.

### **Testing**
Not sure it's testable right now unless you use mock flavor.

### CODE REVIEW

File: app/src/main/java/com/weatherxm/ui/devicealerts/DeviceAlertsActivity.kt [hunk 1]
```diff
@@ -74,10 +74,10 @@ class DeviceAlertsActivity : BaseActivity(), DeviceAlertListener {
...
 
     override fun onLowBatteryReadMoreClicked() {
-        val url = if (device?.profile == DeviceProfile.M5) {
-            getString(R.string.docs_url_low_battery_m5)
-        } else {
-            getString(R.string.docs_url_low_battery_helium)
+        val url = when (device?.profile) {
+            DeviceProfile.M5 -> getString(R.string.docs_url_low_battery_m5)
+            DeviceProfile.D1 -> getString(R.string.docs_url_low_battery_d1)
+            else -> getString(R.string.docs_url_low_battery_helium)
```
Comment (pantelisss) [inline app/src/main/java/com/weatherxm/ui/devicealerts/DeviceAlertsActivity.kt:80]: I would prefer to check explicitly for helium and return an empty string in unhandled cases.

File: app/src/main/java/com/weatherxm/ui/home/devices/DevicesFragment.kt [hunk 2]
```diff
@@ -217,10 +217,10 @@ class DevicesFragment : BaseFragment(), DeviceListener {
...
 
     override fun onLowBatteryReadMoreClicked(device: UIDevice) {
-        val url = if (device.profile == DeviceProfile.M5) {
-            getString(R.string.docs_url_low_battery_m5)
-        } else {
-            getString(R.string.docs_url_low_battery_helium)
+        val url = when (device.profile) {
+            DeviceProfile.M5 -> getString(R.string.docs_url_low_battery_m5)
+            DeviceProfile.D1 -> getString(R.string.docs_url_low_battery_d1)
+            else -> getString(R.string.docs_url_low_battery_helium)
```
Comment (pantelisss) [inline app/src/main/java/com/weatherxm/ui/home/devices/DevicesFragment.kt:223]: same here

**Review by pantelisss — CHANGES_REQUESTED** [review]
Let's discuss about my comments

File: app/src/main/java/com/weatherxm/ui/devicealerts/DeviceAlertsActivity.kt (hunk 1 above)
Comment (PavlosTze) [reply app/src/main/java/com/weatherxm/ui/devicealerts/DeviceAlertsActivity.kt:80]: Valid point, I'll fix it.

**Review by pantelisss — APPROVED** [review]

**Pull request merged**